release-schema.json

```
@@ -11,6 +11,18 @@
           },
           "uniqueItems": true,
           "minItems": 1
+        },
+        "details": {
+          "properties": {
+            "listedOnRegulatedMarket": {
+              "title": "Listed on regulated market",
+              "description": "Whether the organization is listed on a regulated market (e.g. a stock exchange) that ensures adequate transparency of the nationality of beneficial owners.",
```

Here (and in the readme) I don't think the "adequate transparency" is specific to the nationality of beneficial owners. I think it's just (per the regulation) "in line with anti-money laundering legislation" (i.e. nationality transparency is one means to that end, but the regulated market might employ other means).

I'm not entirely sure if the "that ensures" part is an implication of being on a regulated market (in which case we can omit it, since it's not directly describing the semantics) or if it's a condition on which regulated markets count (i.e. I don't know if the EU distinguishes between adequately and inadequately transparent markets).

So, we'll leave it in.

```suggestion
              "description": "Whether the organization is listed on a regulated market (e.g. a stock exchange) that ensures adequate transparency in line with anti-money laundering legislation.",
```

---

> Here (and in the readme) I don't think the "adequate transparency" is specific to the nationality of beneficial owners. I think it's just (per the regulation) "in line with anti-money laundering legislation" (i.e. nationality transparency is one means to that end, but the regulated market might employ other means).

That makes sense. I was trying to capture the relationship between this field and the owner's nationality, but on re-reading the description of the eForms field, I see it's about the publication of the owner's nationality in beneficial ownership registers rather than in eForms, in which the nationality field is [actually mandatory](https://docs.ted.europa.eu/eforms/latest/schema/parties.html#ultimateBeneficialOwnerSection):

>  The nationality (or nationalities) of the beneficiary owner(s) of the winner, tenderer, or subcontractor is not published in the register(s) established by Directive (EU) 2018/843, because the winner is listed on a regulated market (e.g. a stock exchange) that ensures adequate transparency in line with anti-money laundering legislation.

I read the "that ensures" part as a condition since I guess there are (poorly) regulated markets without adequate transparency.